plugins/org.eclipse.thym.ui/src/org/eclipse/thym/ui/config/internal/PropertiesPage.java

```diff
@@ -451,17 +458,14 @@ public void widgetSelected(SelectionEvent e) {
 						.getSelection();
 				if (selection.isEmpty())
 					return;
-				Feature feature = (Feature) selection.getFirstElement();
+				Plugin feature = (Plugin) selection.getFirstElement();
 				IResource resource = (IResource) getEditorInput().getAdapter(IResource.class);
 				HybridProject prj = HybridProject.getHybridProject(resource.getProject());
 				boolean pluginFoundAndRemoved = false;
 				try {
 					List<CordovaPlugin> plugins = prj.getPluginManager().getInstalledPlugins();
 					for (CordovaPlugin cordovaPlugin : plugins) {
-						// This is definitely error prone. As the name for a
-						// plugin is not guaranteed to be unique. Unfortunately feature tag does not have a
-						// reference to plugin id.
-						if (cordovaPlugin.getName() != null && cordovaPlugin.getName().equals(feature.getName())) {
+						if (cordovaPlugin.getId() != null && cordovaPlugin.getId().equals(feature.getName())) {
```

Comment:
Note that the plugin id should be unique, so the issue mentioned in the comment above no longer applies.